src/main/java/edu/ie3/datamodel/io/factory/input/SwitchInputFactory.java

```diff
@@ -34,6 +34,19 @@ protected SwitchInput buildModel(
       OperatorInput operator,
       OperationTime operationTime) {
     final boolean closed = data.getBoolean(CLOSED);
+
+    try {
+      final int parallelDevices = data.getInt(PARALLEL_DEVICES);
+      log.warn(
+          "Found a `SwitchInput` with the id `{}` that specifies `parallelDevices` with a value of `{}`. Because switches cannot be parallel, the given value is ignored!",
+          id,
+          parallelDevices);
+    } catch (Exception ignored) {
+      // because the field should not be used for switches, we can ignore the exception, that is
+      // thrown, when we
+      // do not find the field
+    }
```

Comment:
Suggestion from my side:

1. Check with `data.containsKey` if the value exists, this way we can avoid an exception being thrown
2. If the value exists, we use `data.getField` to retrieve the value because it is going to be placed into the log string anyway. We don't need to parse it beforehand. Plus, if the value is not Integer-parseable as we expect, we still get the warning message.